### PR TITLE
Human Culture Title Localizations

### PR DESCRIPTION
**Pull request by PaperPlate-Github**
## Changelog:
- Special title localizations for Wastewander Culture
- Special title localizations for Kul Tiran Culture
- Special title localizations for Solstice Culture

## Developer changelog:
- Currently only has english localizations.

### CODE REVIEW

File: common/flavorization/00_title_holders.txt [hunk 1]
```diff
@@ -93,6 +93,268 @@ realm_priest_female = {
...
+}
+barony_feudal_wastewander = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	governments = { tribal_government feudal_government clan_government }
+	cultures = { wastewander }
+}
```
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:124]: Barony isn't a character title.

File: common/flavorization/00_title_holders.txt [hunk 2]
```diff
@@ -93,6 +93,268 @@ realm_priest_female = {
...
+count_feudal_female_wastewander = {
+	type = title
+	tier = barony
+	priority = 7
+	ggovernments = { tribal_government feudal_government clan_government }
+	cultures = { wastewander }
+}
```
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:140]: Tier barony?
Type title?
`ggovernments`?

File: common/flavorization/00_title_holders.txt [hunk 3]
```diff
@@ -93,6 +93,268 @@ realm_priest_female = {
...
+	governments = { tribal_government feudal_government clan_government }
+	cultures = { wastewander }
+}
+duke_feudal_female_wastewander = {
+	type = title
+	tier = county
+	priority = 18
+	governments = { tribal_government feudal_government clan_government }
+	cultures = { wastewander }
+}
```
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:165]: Duke character title.

File: common/flavorization/00_title_holders.txt [hunk 4]
```diff
@@ -93,6 +93,268 @@ realm_priest_female = {
...
+duchy_feudal_wastewander = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 18
+	only_independent = yes
+	governments = { tribal_government feudal_government clan_government }
+	cultures = { wastewander }
+}
```
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:175]: Duchy isn't a character title.
Why only `only_independent`?

Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:175]: Tier county?

File: common/flavorization/00_title_holders.txt [hunk 5]
```diff
@@ -93,6 +93,268 @@ realm_priest_female = {
...
+duchy_feudal_northern = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 18
+	only_independent = yes
+	governments = { tribal_government feudal_government clan_government }
+	cultures = { northern }
+}
```
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:254]: Tier county?

File: common/flavorization/00_title_holders.txt [hunk 6]
```diff
@@ -93,6 +93,268 @@ realm_priest_female = {
...
+duke_feudal_female_northern = {
+	type = title
+	tier = county
+	priority = 18
+	governments = { tribal_government feudal_government clan_government }
+	cultures = { northern }
+}
```
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:244]: Duke is character title.

File: common/flavorization/00_title_holders.txt (hunk 5 above)
Comment (zumbak04) [inline common/flavorization/00_title_holders.txt:254]: Duchy isn't a character title.

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: localization/english/culture/wc_culture_titles_l_english.yml [hunk 7]
```diff
@@ -13,3 +13,39 @@
...
+ empire_feudal_northern:0 "Tsardom"
+
+### Kul Tiras ###
+# Feudal, Clan, or Tribal
+ king_feudal_male_tirassian:0 "Lord Admiral"
+ king_feudal_female_tirassian:0 "Lord Admiral"
+ kingdom_feudal_tirassian:0 "Lord Admiralty"
+ emperor_feudal_male_tirassian:1 "Grand Admiral"
+ emperor_feudal_female_tirassian:1 "Grand Admiral"
+ empire_feudal_tirassian:1 "Grand Admiralty"
```
Comment (ElMariuso) [inline localization/english/culture/wc_culture_titles_l_english.yml:51]: Kul'tiras is only named "Admiralty of Kul'tiras" and not "Lord Admiralty of Kul'tiras". So I suggest that it should be "Admiralty of" just for the kingdoms.

image

Comment (zumbak04) [reply localization/english/culture/wc_culture_titles_l_english.yml]: > Kul'tiras is only named "Admiralty of Kul'tiras" and not "Lord Admiralty of Kul'tiras". So I suggest that it should be "Admiralty of" just for the kingdoms.
> 
> image

That's an empire title. So it's a possible title of Kul Tiras if they become more powerful.

**Review by zumbak04 — CHANGES_REQUESTED** [review]
Wait, why was I requested?

**Pull request closed without merging**